### PR TITLE
IBX-6441 - Add post visit and price drop

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | [IBX-6441](https://issues.ibexa.co/browse/IBX-6441) |
| Versions      | 3.3 and up |
| Edition       | all |

Document additional triggers and new events.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/personalization/api_reference/tracking_api.md [hunk 1]
```diff
@@ -190,23 +190,99 @@ For a list of embedded parameters that each of the events may use, see the follo
 |`sourceuserid`|User identifier valid up to now(usually some anonymous session ID)|URL-encoded alphanumeric|
 |`targetuserid`|User identifier valid from now on (usually an account ID or login name)|URL-encoded alphanumeric|
 
+### Basket event
```
Comment (nkostiv) [inline docs/personalization/api_reference/tracking_api.md:193]: Why it was moved before click event?

Comment (dabrt) [reply docs/personalization/api_reference/tracking_api.md:193]: The list is now arranged alphabetically

File: docs/personalization/api_reference/tracking_api.md [hunk 2]
```diff
@@ -237,85 +313,63 @@ The following table lists the request parameters:
...
-revenue for statistics. 
-The price must be a price the user paid for the item, including all taxes and discounts. 
+This event is issued when the end user removes items from their wishlist.
+It could signify that the user has lost interest in the product.
+Based on this information, recommendations presented by the store can be more accurate.
 
-If product price filtering is activated, the information provided over the product 
-import is used. 
-For more information, see [Content API](content_api.md).
+`GET https://event.perso.ibexa.co/api/[customerid]/deletefromwishlist/[userid]/[itemtypeid]/[itemid]?`
```
Comment (nkostiv) [inline docs/personalization/api_reference/tracking_api.md:353]: ? symbol at the end should not be as we don't have any parameters

**Review by nkostiv — APPROVED** [review]

**Review by julitafalcondusza — APPROVED** [review]

**Review by MagdalenaZuba — APPROVED** [review]

**Comment by MagdalenaZuba** [comment]
Please ensure consistent usage of 'Buy events' or 'buy events' throughout the page for uniformity.

**Pull request merged**